### PR TITLE
enhancement/flake8

### PR DESCRIPTION
**Pull request by losete**
Pull Request made for applying flake8 style. Additionally, this tests will be added to Jenkins

### CODE REVIEW

**Review by Chusca — APPROVED** [review]

**Review by sebascm — APPROVED** [review]

**Review by pablogalegoc — APPROVED** [review]

File: test.py [hunk 1]
```diff
@@ -13,8 +13,9 @@ def test_father(self):
     def test_mother(self):
         user = '__'
         tweets = list(get_tweets(query=user, pages=1))
+        text = 'It is a gift to be alive in the time of Beyoncé'
 
-        self.assertTrue(tweets[0]['text'].__contains__('It is a gift to be alive in the time of Beyoncé'))
+        self.assertTrue(tweets[0]['text'].__contains__(text))
```
Comment (franrua) [inline test.py:18]: Are these changes necessary for flake8 checkstyle?

**Review by franrua — COMMENTED** [review]
Only a commnet, no changes necessary
